Cargo.toml

```diff
@@ -124,9 +124,10 @@ tendermint-config = "0.33.0"
 tendermint-rpc = { version = "0.31", features = ["secp256k1", "http-client", "websocket-client"] }
 tendermint-proto = { version = "0.31" }
 
-ipc-sdk = { git = "https://github.com/consensus-shipyard/ipc-agent.git", branch = "dev" }
+ipc-sdk = { git = "https://github.com/consensus-shipyard/ipc-agent.git", branch = "validator-metadata-change" }
```

Comment:
will update to `dev` once merged the depending PRs.

Reply:
Changed to `dev` after rebasing from `main`